README.md

```diff
@@ -1,3 +1,21 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+###
+#### A 1-2 summary or bullet points outlining your plan for check-ins throughout the duration of the project.
+   Every morning, we are doing a stand-up meeting before class to check in about the project, give feedbacks and update each other.
+   Meet Friday morning to retro 
+
```

Comment:
Maybe we can add topics around things we struggled with when building out our assigned methods